src/core/App.js

```diff
@@ -28,17 +28,24 @@ class App extends ModuleSystem {
     })();
 
     function process() {
-      requestAnimFrame(process.bind(this));
+      if (this.render) {
```

Comment:
`render` has quite different meaning and is more related to `RenderingModule` than `App`. How about renaming it to `update` or `updateEnabled`?